src/perspective_view/panel.ts

```diff
@@ -856,10 +987,70 @@ export class PerspectivePanel extends RenderedDataPanel {
         if (renderLayer.isTransparent) {
           renderContext.depthBufferTexture =
             this.offscreenFramebuffer.colorBuffers[OffscreenTextures.Z].texture;
+        }
+        const isVolumeRenderingLayer = "mode" in renderLayer;
```

Comment:
I guess this may be to avoid a circular dependency but I think a less hacky approach would be better here.

Reply:
Thanks Jeremy, I did have this to avoid a circular dependency, but I agree that should be changed. I've used the `isVolumeRendering` getter on the layer itself after review, let me know if that's better. Happy to look at it again if not!